proto/v1alpha1/rode_policy.proto

```diff
@@ -224,3 +224,38 @@ message PolicyEvaluation {
   // Violations is a list of rule results. Even if a rule passed, its output will be included in Violations.
   repeated EvaluatePolicyViolation violations = 5;
 }
+
+// PolicyAssignment represents a link between a policy and a policy group. Assignments can only be done against policy versions, to allow
+// for a safe rollout of new changes.
+message PolicyAssignment {
+  // Id uniquely identifies the policy assignment. It is also a path that can be used to retrieve, update or delete the assignment.
+  string id = 1;
+  // PolicyVersionId is the unique identifier (UUID) of a PolicyEntity. The version must exist at the time of assignment.
+  string policy_version_id = 2;
+  // PolicyGroup is corresponds to PolicyGroup.Name. The group must exist at the time of creation.
```

Comment:
Not entirely sure how I feel about this, but one thing I liked about grafeas' protos is that their comments would say things like "output only" or "required", which helped me understand the lifecycle of their models a bit better.  in this case, it might be a good idea to mention how this field isn't updatable after a `PolicyAssignment` is created.  obviously, there are a lot of other places where we'd probably have to do this too.

what do you think?

Reply:
I like including "required" and "immutable" or "output-only" in the comments. I can add that to this proto and then make an issue.

Reply:
Opened https://github.com/rode/rode/issues/114